ruby-training/trainees/shareni/test_katas/05/sum_array/array_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rspec'
+require_relative 'new_array'
+
+describe "Array" do
+    describe "#sum" do
+        it "returns the sum of the array without the maximum and minimum value" do
+           example_array = NewArray.new([1,2,8,10])
+           result = example_array.sum_without_max_and_min
+           expect(result).to eq(10)
+        end
+
+        it "returns the sum of the array without one minimum if there are more than one minimums" do
```

Comment:
Your "it" block descriptions are pretty good. I can only suggest using suggestive language rather than definitive language ("should return" rather than "returns"). In the case of documenting code through specs, we should remember that the description's purpose is to let us know what the code "should" be doing so we can compare that to the actual result and see if anything is off. Now, this is super nitpicky and unlikely, but if we use "returns" it may give a less experienced developer the subconscious idea that everything is working fine and it may lead to errors down the road.

Finally, also nitpicky but ultimately important. The proper grammar in this case, also considering the statements above, would be:

`it "should return the sum of the array without one minimum if there is more than one minimum" do` ...